projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-wizard/dossier-management-widget-wizard.component.html

```diff
@@ -82,14 +82,16 @@ <h3 cdsModalHeaderHeading>
       {{ 'interface.cancel' | translate }}
     </button>
 
-    <button
-      class="valtimo-dossier-management-widget-wizard__back"
-      cdsButton="secondary"
-      [disabled]="backButtonDisabled()"
-      (click)="onBackButtonClick()"
-    >
-      {{ 'interface.back' | translate }}
-    </button>
+    @if (currentStep() !== WidgetWizardSteps.TYPE) {
+      <button
+        class="valtimo-dossier-management-widget-wizard__back"
+        cdsButton="secondary"
+        [disabled]="backButtonDisabled()"
```

Comment:
backButtonDisabled() signal needs to be updated to remove the currentStep condition